index.js

```diff
@@ -20,11 +27,20 @@ module.exports = function (req) {
       var opts = options || {};
       opts.expires = new Date(0);
 
-      return !!(req.res.cookie(name, '', opts));
+      if (req) {
+          req.res.cookie(name, '', opts)
+      }
+
+      try {
+          delete cookiesSet[name];
```

Comment:
if there's an error trying to `delete` i guess i could just `return !!(delete cookiesSet[name])`, i was just trying to say "if it failed to delete for any reason, return false"